vignettes/04-synthesis-data.Rmd

```diff
@@ -137,53 +136,21 @@ Below we retrieve all the available plots for a particular date, then find and c
 We will use these tuples to extract the data for our plot.
 
 ```{r get_plot_boundary}
-library(traits)
-library(stringr)
-
-# Function for breaking apart a corner into its Lat, Lon components
-getLatLon <- function(corner){
-  p <- strsplit(corner, ' ')
-  return (c(p[[1]][1], p[[1]][2]))
-}
-
-# Gets the bounding box of the array of points
-getBounds <-function(bounds){
-  minX <- NA
-  minY <- NA
-  maxX <- NA
-  maxY <- NA
-  for (c in unique(bounds)){
-    p = getLatLon(c)
-    if (is.na(minX) || (minX > p[2]))
-      minX = p[2]
-    if (is.na(minY) || (minY > p[1]))
-      minY = p[1]
-    if (is.na(maxX) || (maxX < p[2]))
-      maxX = p[2]
-    if (is.na(maxY) || (maxY < p[1]))
-      maxY = p[1]
-  }
-  
-  return (c(minX, minY, maxX, maxY))
-}
-
 # Setting up our options
 options(betydb_url = "https://terraref.ncsa.illinois.edu/bety/",
         betydb_api_version = 'v1')
 
-# Makiong the query for our site
+# Making the query for our site
 sites <- betydb_query(table     = "sites",  
                       sitename  = "MAC Field Scanner Season 6 Range 19 Column 1")
 
 # Assigning the geometry of the site (GeoJSON format)
 site.geom <- sites$geometry
 
-# Stripping out the extra information to get to the points
-complete_str <- str_match_all(site.geom, '(\\(\\(\\((.*)\\)\\)\\))')[[1]][, 3]
-bounds <- strsplit(complete_str, ', ')[[1]]
-
-# Getting the bounding box of the polygon
-bounding_box = getBounds(bounds)
+# Convert the polygon to something we can clip with. CRS value represents WGS84 Lat/Long
+site.shape = st_as_sfc(site.geom,crs = 4326)
```

Comment:
R style convention: use <- not = for assignment

Reply:
The text is in there. I've broken it out onto its own line to be more visible.